scripts/run_e2e_tests.py

```diff
@@ -168,7 +168,7 @@ def cleanup():
     """Kill the running subprocesses and server fired in this program, set
     constants back to default values.
     """
-    google_app_engine_path = '%s/' % common.GOOGLE_APP_ENGINE_HOME
+    google_app_engine_path = '%s/' % common.GOOGLE_CLOUD_SDK_BIN
```

Comment:
Should we change the variable name `google_app_engine_path` similarly?

Reply:
It is actually still the google app engine path just in a different location.